laws/shared/src/test/scala/algebra/laws/LawTests.scala

```diff
@@ -143,5 +143,32 @@ trait LawTestsBase extends FunSuite with Discipline {
     implicit val longSemilattice: Semilattice[Long] = LongMinMaxLattice.joinSemilattice
     laws[LatticeLaws, (Set[Int], Long)].check(_.semilattice(lexicographicSemilattice))
   }
+
+  {
+    // This "arbitrary int order" isn't that arbitrary.
+    // I think this could be improved once we have a version of Scalacheck with Cogen.
+    implicit val arbOrderInt: Arbitrary[Order[Int]] = Arbitrary(
+      Gen.oneOf(
+        Gen.const(Order[Int]),
+        Gen.const(Order[Int].reverse)))
+
+    // This is a hack to fake an `Eq` instance for an `Order`.
```

Comment:
As an alternative (though this is fine), you could also just literally fix the input type to only contain some small, fixed # of elements (instead of `Int`). Then we could just sort 2 lists using 2 different orders to check equality.

```scala
case class N100(n: Int) { require(n >= 0 && n < 100) }
implicit val arbN100 = Arbitrary(Gen.choose(0, 100).map(N100))
implicit val argN100Order = arbitrary[Int].map { seed =>
  val order = new Random(seed).shuffle(Vector.range(0, 100))
  Order.by { (n: N100) => order(n.n) }
}
implicit val N100OrderEq: Eq[Order[N100]] = Eq.by { implicit order => Vector.tabulate(100)(N100).sorted.map(_.n) }
```